WebAPIClient/WebAPIClient/Repository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+
+namespace WebAPIClient
+{
+    /// <summary>
+    /// Class to hold objects received via JSON - repos in this case
+    /// </summary>
+    public class Repository
+    {
+        [JsonPropertyName("name")]
```

Comment:
We will commonly not do this and set the JSON deserializer to be case insensitive

```cs
var options = new JsonSerializerOptions
{
    PropertyNameCaseInsensitive = true
};
var weatherForecast = JsonSerializer.Deserialize<WeatherForecast>(jsonString, options);
```